subprojects/soak/src/testFixtures/groovy/org/gradle/launcher/daemon/fixtures/DaemonMultiJdkIntegrationTest.groovy

```diff
@@ -50,18 +50,24 @@ class DaemonMultiJdkIntegrationTest extends DaemonIntegrationSpec {
                 @Override
                 boolean isSatisfiedBy(JvmInstallationMetadata install) {
                     if (version.hasProperty("vendor")) {
-                        if(install.getVendor().getKnownVendor() != version.vendor) {
+                        def actualVendor = install.getVendor().getKnownVendor()
+                        def expectedVendor = version.vendor
+                        if (actualVendor != expectedVendor) {
                             return false
                         }
                     }
-                    return install.languageVersion == version.version
```

Comment:
This was always false, since the types were different, so the soak test above never ran.